cluster/gce/gci/configure-helper.sh

```diff
@@ -143,7 +143,9 @@ function main() {
     start-lb-controller
     update-legacy-addon-node-labels &
     apply-encryption-config &
-    start-cluster-networking   ####start cluster networking if not using default kubenet
+    if [[ "${ARKTOS_SCALEOUT_SERVER_TYPE:-}" != "rp" ]]; then
```

Comment:
it would be clearer to use ARKTOS_SCALEOUT_SERVER_TYPE in desiredTypes, to start the container network explicitly on the needed servers, currently the ARKTOS_SCALEOUT_SERVER_TYPE can be node, tp, rp, proxy and this might be expand as well. this way,  some unwanted calls can be avoided here.

Reply:
updated in new commit